docs/Course Wiki/ECE Course Offerings/ECE340.md

```diff
@@ -32,23 +31,23 @@ Relevant topics from ECE329 include Maxwell's equations in material media and ca
 
-Homework assignments are assigned each week and usually consist of three or four problems that may involve drawing band diagrams, derivations, calculations, conceptual explanations, or using an online semiconductor simulation applet. There will be multiple times where certain assumptions must be made (temperature independence, neglecting a small concentration, etc) to make ceratin problems solable in a reasonable amount of time. This can be frustrating for students used to solving problems with the exact parameters given to them, but making and justifying these assumptions are part of the course.
+Homework assignments are assigned each week and usually consist of three or four problems that may involve drawing band diagrams, derivations, calculations, conceptual explanations, or using an online semiconductor simulation applet. There will be multiple times where certain assumptions must be made (temperature independence, neglecting a small concentration, etc) to make certain problems solvable in a reasonable amount of time. This can be frustrating for students used to solving problems with the exact parameters given to them, but making and justifying these assumptions are part of the course.
 
-Midterm exams for ECE340 are quite imfamous among the 300 level ECE courses. While slight logistical things can vary between semesters (you may or may not see the exam equation sheet before the midterm), in general the tests are 3 questions with several subparts to be solved in 1 hour without a calculator. They are usually even more of a test of time management than ECE210 exams. The strangest quirk about this course is the lack of access to past exams. The testing format also means that there is little similarity to homework problems. This structure makes understanding the concepts taught in ECE340 paramount to success.
+Midterm exams for ECE340 are quite infamous among the 300 level ECE courses. While slight logistical things can vary between semesters (you may or may not see the exam equation sheet before the midterm), in general the tests are 3 questions with several subparts to be solved in 1 hour without a calculator. They are usually even more of a test of time management than ECE210 exams. The strangest quirk about this course is the lack of access to past exams. The testing format also means that there is little similarity to homework problems. This structure makes understanding the concepts taught in ECE340 paramount to success.
 
 ## Instructors
 
-Prof. Leburton is the course director. Professors that have taught ECE 340 in the past include Prof. Rakheja, Dragic, He, Zhu, Dallesasse, Bogdanov to name a few. Most professors who teach this course specialize in device physics or nanoelectronics.
+Prof. Leburton is the course director. Professors that have taught ECE 340 in the past include Prof. Rakheja, Dragic, He, Zhu, Dallesasse, Bogdanov to name a few. Most professors who teach this course specialize in device physics of nanoelectronics.
```

Comment:
Shouldn't it be "device physics or nanoelectronics"? Otherwise, should it be "the device physics of nanoelectronics"?